src/main/java/org/myrobotlab/service/Servo.java

```diff
@@ -128,6 +128,9 @@ protected boolean processMove(Double newPos, boolean blocking, Long timeoutMs) {
       log.info("{} is currently blocking - ignoring request to moveTo({})", getName(), newPos);
       return false;
     }
+    
+    broadcast("publishServoMoveTo", getTargetOutput());

```

Comment:
This should be passing a servoMove object,... and why is this the output ? shouldn't it be publishing the input ?

Reply:
current method signature in AbstractServo
`
  @Override
  public ServoMove publishServoMoveTo(Double pos) {
    log.debug("{}.publishServoMoveTo({})", getName(), pos);
    return new ServoMove(getName(), pos);
  }
`
could make it symmetrical, but not all interfaces adhere to that ... :shrug: - don't have much opinion

To answer the more significant question I think,
is this input or output - This method was made to be consumed by the ServoController ...
And it always has been output.  ie.  the ServoController.onServoMoveTo requires output - but potentially i can find this itself by using the service name to get the service to get the Servo.getTargetOutput()

... its a good question :shrug: